Userland/Libraries/LibCrypto/BigInt/SignedBigInteger.cpp

```diff
@@ -109,11 +109,15 @@ FLATTEN SignedBigInteger SignedBigInteger::minus(const SignedBigInteger& other)
     // -x - -y = y - x
     if (m_unsigned_data < other.m_unsigned_data) {
         // The result will be positive.
-        return SignedBigInteger { other.m_unsigned_data.minus(m_unsigned_data), true };
+        return SignedBigInteger { other.m_unsigned_data.minus(m_unsigned_data) };
```

Comment:
lol comment goes "the result will be positive" and then sets it to negative
classic